src/index.tsx

```diff
@@ -1,13 +1,13 @@
 import React, { StrictMode } from 'react';
 import ReactDOM from 'react-dom';
-import App from 'App';
 import * as serviceWorker from 'serviceWorker';
 
 import 'styles/styles.scss';
+import { AppRouter } from 'routes/AppRouter';
 
 ReactDOM.render(
   <StrictMode>
-    <App />
+    <AppRouter />
```

Comment:
i don't know if AppRouter is good because they contains more that router match

Reply:
No worries, this is an example. Later when everything change the Navbar will be move to his corresponding component.

Reply:
ok